src/main/java/org/assertj/core/api/AbstractMapAssert.java

```diff
@@ -99,6 +99,39 @@ public SELF allSatisfy(BiConsumer<? super K, ? super V> entryRequirements) {
     return myself;
   }
 
+  /**
+   * Verifies that at least one map entry satisfies the given {@code entryRequirements} .
+   * <p>
+   * Example:
+   * <pre><code class='java'> Map&lt;TolkienCharacter, Ring&gt; elvesRingBearers = new HashMap&lt;&gt;();
+   * elvesRingBearers.put(galadriel, nenya);
+   * elvesRingBearers.put(gandalf, narya);
+   * elvesRingBearers.put(elrond, vilya);
+   *
+   * // this assertion succeeds:
+   * assertThat(elvesRingBearers).anySatisfy((character, ring) -&gt; {
+   *   assertThat(ring).isIn(vilya);
+   * });
+   *
+   * // this assertion fails as Gandalf the maia does not bear the ring vilya and the ring bearer of vilya is not a maia:
+   * assertThat(elvesRingBearers).anySatisfy((character, ring) -&gt; {
+   *   assertThat(character.getRace()).isIn(MAIA);
+   *   assertThat(ring).isIn(vilya);
```

Comment:
same comment here

Reply:
Consequently (with the upper proposal) I would add `isEqualTo(oneRing)`. While the tuple (maia,vilya) cannot be satisfied in the tolkien universe, the tuple (maia, oneRing) can - but not by the elven ring bearers.